src/main/java/org/broadinstitute/hellbender/utils/bwa/BwaMemAlignmentUtils.java

```diff
@@ -88,6 +88,7 @@ else if ( alwaysGenerateASandXS ) {
             samRecord.setReadBases(SAMRecord.NULL_SEQUENCE);
             samRecord.setBaseQualities(SAMRecord.NULL_QUALS);
         }
+        //TODO: there ought to be a way to indicate a set of tag names that ought to be copied -- we're just doing RG
         if ( readGroup != null ) samRecord.setAttribute(SAMTag.RG.name(), readGroup);
```

Comment:
Add curly braces around if statement body.

Reply:
done, although I didn't hit the rest of the file